swarm/network/stream/peer.go

```diff
@@ -77,9 +78,10 @@ type WrappedPriorityMsg struct {
 }
 
 // NewPeer is the constructor for Peer
-func NewPeer(peer *protocols.Peer, streamer *Registry) *Peer {
+func NewPeer(peer *network.BzzPeer, streamer *Registry) *Peer {
 	p := &Peer{
-		Peer:         peer,
+		Peer:         peer.Peer,
```

Comment:
Explicitly keeping a pointer to peer.Peer seems redundant.

Reply:
See above